templates/tools/dockerfile/interoptest/grpc_interop_java/Dockerfile.include

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-<%include file="../../debian_8_header.include"/>
+FROM debian:jessie
```

Comment:
I agree that using debian 8 as opposed to debian jessie might make sense. But currently most references use the distro codename (jessie/stretch/buster/bullseye) and not the version number, and we really should be consistent.
So to achieve consistency without causing too much churn, I decided to unify everything on using the debian codenames and not numbers.

We could change everything from debian codenames to version numbers, but it should be done as a separate PR (since it's basically a refactoring and it's going to be much easier to review that way).